src/Period/LargePeriodPlainTextFormatter.php

```diff
@@ -10,9 +10,16 @@
 
 use \CultureFeed_Cdb_Data_Calendar_SchemeDay as SchemeDay;
 use IntlDateFormatter;
+use phpDocumentor\Reflection\Types\This;
 
 class LargePeriodPlainTextFormatter implements PeriodFormatterInterface
 {
+    private $locale;
+
+    public function __construct($locale)
```

Comment:
Old default value is missing here, this would be a BC break causing issues for existing uses of the library.